inc/snowball-options.php

```diff
@@ -0,0 +1,13 @@
+<div class="wrap">
+	<h2>Snowball Options</h2>
```

Comment:
Call this "Snowball Settings" to keep it consistent.

Fix tabs to 2 spaces.

Add newline at end of file.